src/Types/Workspace/Builder.php

```diff
@@ -139,7 +139,7 @@ public function build(Environment $environment, DefinitionCollection $definition
                 ->usage('config dump --key=<key>')
                 ->action(function (Input $input) use ($environment) {
                     $key = $input->getOption('key');
-                    $key = $key instanceof OptionValue ? $key->value() : $key;
+                    $key = $key->value();
                     $attribute = $environment->getAttribute($key);
```

Comment:
This is reverting a fix

Reply:
`getOption` can only return an OptionValue unless I miss something:

https://github.com/my127/php-library/blob/master/src/Console/src/Usage/Input.php#L85

Reply:
```
    public function getOption($option): OptionValue
```

Reply:
It was https://github.com/my127/php-library/commit/60e14a537fd2898e6f5a364394c6aee3dd9b9fd1 I was thinking of, which changed `Input::option()` rather than `Input::getOption()`